index.js

```diff
@@ -501,7 +501,13 @@ class LedgerBridgeKeyring extends EventEmitter {
 
       return undefined
     }
-    window.addEventListener('message', eventListener)
+    window.addEventListener('message', this._eventListener)
+  }
+
+  destroy () {
+    window.removeEventListener('message', this._eventListener)
+    this.currentMessageId = 0
+    this.messageCallbacks = {}
```

Comment:
Removing the event listener makes sense as a cleanup.  I don't believe that we need to reset the other two variables, as destroying this keyring and instantiating a new one would do that within the constructor.